src/api/use-accept-language.ts

```diff
@@ -0,0 +1,8 @@
+import {useTranslation} from '@atb/translations';
+
+export const useAcceptLanguage = () => {
```

Comment:
A different approach which can be considered, is to add a useEffect which updates the `Accept-Language` axios default header when `language` changes. However this would affect all axios requests, unless we also add an option for a handler in `createClient`.

Another difference would be that since `acceptLanguage` would not be in the query key, it will not be refetched when `language` is changed.

So for now at least I think it is easiest to use this hook instead.